test/tasks_screen/models/task_database_operations_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:lachies_life_planner/shared/global_config.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task-database-operations.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task.dart';
+
+import '../../utils/mock_firestore_data.dart';
+
+void main() {
+  group('Task', () {
+    MockFirestoreInstance mockFirestoreInstance;
+
+    setUp(() {
+      mockFirestoreInstance = MockFirestoreInstance();
+      setFirestoreInstance(mockFirestoreInstance);
```

Comment:
Can do `setFirestoreInstance(MockFirestoreInstance())` and then just use `getFirestoreInstance()`

Reply:
Done